index.bs

```diff
@@ -320,8 +322,15 @@ Issue: Include cross references to the specs for these hash functions.
     value.  This signals to the listening agent that it should connect to the
     advertising agent to discover updated metadata.
 
-Issue: Add examples of sample mDNS records.
+The advertising agent may add an additional field to the TXT record:
+
+: pw
+:: An alphanumeric, unguessable token consisting of a [=Base64=] encoded 128-bit
+    random value (24 characters in length).
```

Comment:
Why not just say a random alphanumeric string?  Why specify it has to be Base64?

Reply:
It can't be any "random string".   It needs to be unguessable and have some minimum quantity of true entropy.  And it needs to be encoded efficiently to not waste bytes in the TXT.

I removed the base64 requirement though.  The specific encoding isn't important.